apis/Google.Devtools.AspNet/Google.Devtools.AspNet/CloudTrace/IManagedTracer.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Google.Devtools.AspNet
+{
+    /// <summary>
+    /// Mangages creating spans for a trace as well as adding meta data to them.
+    /// </summary>
+    public interface IManagedTracer
+    {
+        /// <summary>
+        /// Starts a new span using the most recent (if any) unfinished span as the parent.
+        /// </summary>
+        /// <param name="name">The name of the span.</param>
```

Comment:
No this cannot be null and is required by the API.

The Span options just allow setting the span kind (defaults to unspecified) but will allow me to add other options in the future.